simple-wp-helpdesk/admin/class-reporting.php

```
@@ -239,3 +242,50 @@ function swh_report_first_response_time() {
+
+	$total = (int) ( new WP_Query(
+		array(
+			'post_type'      => 'helpdesk_ticket',
+			'post_status'    => 'publish',
+			'posts_per_page' => -1,
+			'fields'         => 'ids',
+		)
+	) )->found_posts;
+
+	// phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query
+	$open = (int) ( new WP_Query(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '266', column '22'). (undefined)

(MissingImport)